### PR TITLE
feat: simplify formatHours to always show hours only

### PR DESCRIPTION
**Pull request by pepicrft**
Needed for https://github.com/tuist/tuist/pull/7807#issuecomment-3062647556

## Summary
Simplifies the `formatHours` function to always display time as hours only, removing the day/hour breakdown for a cleaner, more consistent format.

## Changes
- Always formats as "Xh" regardless of duration (e.g., "1h", "25h", "168h")
- Rounds fractional hours to nearest whole hour using `Math.round()`
- Simplified implementation with consistent behavior across all time ranges
- Removed complex day/hour calculations for better readability
- Updated all tests to match hours-only format

## Examples
- `formatHours(1.5)` → `"2h"`
- `formatHours(25.5)` → `"26h"` 
- `formatHours(168)` → `"168h"` (1 week)
- `formatHours(720)` → `"720h"` (30 days)

## Test plan
- [x] All existing tests updated and passing
- [x] Verified edge cases like negative values and decimal precision
- [x] Confirmed proper rounding behavior for all fractional hours
- [x] Validated consistent "Xh" format across all time ranges

### CODE REVIEW

File: web/js/formatters.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
 /**
  * Formats hours into a human readable string
  * @param {number} hours - The time duration in hours
- * @returns {string} Formatted time string (e.g., "30m", "2h 30m", "1d 5h")
+ * @returns {string} Formatted time string (e.g., "1h", "25h", "168h")
  */
 export function formatHours(hours) {
-  if (hours < 1) {
```
[the remark this answers is not in the document]
Comment (fortmarek) [reply web/js/formatters.js:7]: I see. I still think this will lead to weird values if all the ticks are less than one hour (typically for freshly created projects). Otherwise, I'm good with the changes.

[the remark this answers is not in the document]
Comment (pepicrft) [reply web/js/formatters.js:7]: If they are under an hour, we pick another formatter that shows minutes.
